app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def mostrecentcomments
+    comments.order(created_at: :desc).limit(5)
+  end
+  after_create :incrementpostcounter
+
+  private
+
+  def incrementpostcounter
+    author.increment!(:post_counter)
+  end
```

Comment:
- - Screenshot reference:
![Selection_017](https://github.com/maths29/iwuBlog/assets/107632780/53a9ecd1-cc3c-4b46-a935-be2a422cf141)

- [ ] I noticed that your methods and function names don't follow the `Ruby best practices` of using `snake_case` for method names. it's important to follow the convention of using `snake_case` for method and function names in Ruby. This convention enhances code readability and consistency, making it easier for the team to collaborate and maintain the code. Please consider updating all your method names to follow the required Ruby best practices as stated in the project requirement. EG: `increment_post_counter` :+1: :wave: :smile: :small_airplane: